readme.md

```diff
@@ -66,14 +87,12 @@ $ strip-css-comments --help
     $ strip-css-comments < src/app.css --all
 ```
 
-
 ## Benchmark
 
 ```
 $ npm run bench
 ```
 
-
```

Comment:
You're right. Thanks! 

I've returned these spaces and after comparing `readme.md` on this branch with the master branch, I've deleted one line and added another.